docs/azure-saas-docs/content/en/components/Identity/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+type: docs
+title: "Identity Framework"
+linkTitle: "Identity Framework"
+weight: 30
+---
+
+The goal of our identity and authorization strategy is to enable us to easily authenticate users and provide basic RBAC for entities created within the application.
+
+> Recommended Reading:
+> * [What is Azure Active Directory B2C?](https://docs.microsoft.com/en-us/azure/active-directory-b2c/overview)
+> * [Architect multitenant solutions on Azure](http://aka.ms/multitenancy)
+
+## Overview
+
+Our Identity Framework is comprised of two main pieces:
+
+1. Identity Provider (IDP) - Preforms user login and authorization and provides a JWT token to the web applications. ASDK comes with Azure AD B2C implemented as the IDP out of the box.
+
+2. Permissions Service - A microservice that tracks what data each user has access to, and serves as an endpoint for the IDP to enrich the user's token with permissions and role claims.
+
+The Identity Framework also has a dependency on the [Microsoft Graph API](https://docs.microsoft.com/en-us/graph/overview), which we use to look up certain user information when needed.
```

Comment:
Some expansion on the scenarios where this user data would be necessary would be helpful here.